Template/NLO/SubProcesses/reweight_xsec_events.f

```diff
@@ -499,57 +519,150 @@ subroutine reweight_scale_ext
       return
       end
 
-      
+!<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<AntonS      

```

Comment:
Just a style detail
avoid to use "<<<<<<<" since those are use for marking conflict by git (or other)